views/macros/navigation.nunj

```diff
@@ -14,9 +14,9 @@
         {% if item.isCollection or (item.isComponent and not item.isCollated and item.variants().filter('isHidden', false).size > 1) %}
         <li class="Tree-item Tree-collection Tree-depth-{{ depth }}" data-behaviour="collection" id="tree-{{ root.name }}-collection-{{ item.handle }}">
             <h4 class="Tree-collectionLabel" data-role="toggle">
-                <span>{{ item.label }}</span>
+                <button aria-expanded="false" aria-controls="tree-{{ root.name }}-collection-{{ item.handle }}-items">{{ item.label }}</button>
```

Comment:
the `aria-expanded="false"` attribute is wrong i think since the collections are closed with js on page load, so this should default to `true` here and be switched when the collection is closed.

Reply:
Makes sense!